frontend/src/main/resources/puf/frisbee/frontend/view/LevelView.fxml

```diff
@@ -207,4 +208,8 @@ angepasst werden, um Fehlermeldungen in Eclipse zu vermeiden. -->
          </children>
       </Group>
    </children>
+   <stylesheets>
+      <URL value="@../css/main.css" />
+      <URL value="@../css/level.css" />
```

Comment:
Dann kann das `scene.getStylesheets().add(getClass().getResource("/puf/frisbee/frontend/css/level.css").toExternalForm());` im `ViewHandler` raus, oder? Vllt können wir generell im `ViewHandler` die main.css hinzufügen (ich weiss nicht, ob man das vllt irgendwie generisch für jede view machen kann) und die spezifischen dann in der .fxml-Datei? Oder ist das zu inkonsistent?